sdcm/nemesis.py

```diff
@@ -4020,7 +4020,10 @@ def disrupt_add_remove_dc(self) -> None:
                     strategy.replication_factors.update({new_dc_name: 1})
                     replication_strategy_setter(**{keyspace: strategy})
                 InfoEvent(message='execute rebuild on new datacenter').publish()
-                new_node.run_nodetool(sub_cmd=f"rebuild -- {datacenters[0]}", retry=0)
+                with wait_for_log_lines(node=new_node, start_line_patterns=["rebuild.*started with keyspaces="],
```

Comment:
need to think/see how this fits with the `adaptive_timeout` or soft timeout

Reply:
still we we should be able to use both context managers I think - and `adaptive_timeout` yielded value could be used here in `end_timeout`.